src/components/application_manager/src/resumption/resumption_data_processor_impl.cc

```diff
@@ -881,7 +881,7 @@ void ResumptionDataProcessorImpl::DeleteButtonsSubscriptions(
   SDL_LOG_AUTO_TRACE();
   const ButtonSubscriptions button_subscriptions =
       application->SubscribedButtons().GetData();
-  for (auto& btn : button_subscriptions) {
+  for (auto btn : button_subscriptions) {
```

Comment:
```suggestion
  for (const auto& btn : button_subscriptions) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e